src/test/java/ru/training/at/hw2/ex1/Exercise1.java

```diff
@@ -0,0 +1,204 @@
+        softAssertObject.assertEquals(serviceButton.getText(), "SERVICE");
+        softAssertObject.assertEquals(metalsAndColorsButton.getText(), "METALS & COLORS");
+    }
+
+    public void checkIndexPageIcons() {
+        //Locating elements with CSS selectors
+        WebElement microscopeIcon = webDriver.findElement(By.cssSelector("div.benefit-icon>span.icons-benefit.icon-practise"));
+        WebElement headsetIcon = webDriver.findElement(By.cssSelector("div.benefit-icon>span.icons-benefit.icon-custom"));
+        WebElement screenIcon = webDriver.findElement(By.cssSelector("div.benefit-icon>span.icons-benefit.icon-multi"));
+        WebElement rocketIcon = webDriver.findElement(By.cssSelector("div.benefit-icon>span.icons-benefit.icon-base"));
+
+        //SoftAsserting that the icons are displayed
+        softAssertObject.assertTrue(microscopeIcon.isDisplayed());
+        softAssertObject.assertTrue(headsetIcon.isDisplayed());
+        softAssertObject.assertTrue(screenIcon.isDisplayed());
+        softAssertObject.assertTrue(rocketIcon.isDisplayed());
+    }
+
+    public void checkTextUnderIcons() {
+        //Expected texts
+        String expectedTextMicroscope = "To include good practices\nand ideas from successful\nEPAM project";
+        String expectedTextHeadset = "To be flexible and\ncustomizable";
+        String expectedTextScreen = "To be multiplatform";
+        String expectedTextRocket = "Already have good base\n(about 20 internal and\nsome external projects),\nwish to get more…";
+
+        //Locating the elements with CSS selectors
+        WebElement microscopeText = webDriver.findElement(By.cssSelector("div.row.clerafix.benefits>div:nth-child(1).col-sm-3>div.benefit>span.benefit-txt"));
+        WebElement headsetText = webDriver.findElement(By.cssSelector("div.row.clerafix.benefits>div:nth-child(2).col-sm-3>div.benefit>span.benefit-txt"));
+        WebElement screenText = webDriver.findElement(By.cssSelector("div.row.clerafix.benefits>div:nth-child(3).col-sm-3>div.benefit>span.benefit-txt"));
+        WebElement rocketText = webDriver.findElement(By.cssSelector("div.row.clerafix.benefits>div:nth-child(4).col-sm-3>div.benefit>span.benefit-txt"));
+
+        //SoftAsserting that the texts are displayed
+        softAssertObject.assertTrue(microscopeText.isDisplayed());
+        softAssertObject.assertTrue(headsetText.isDisplayed());
+        softAssertObject.assertTrue(screenText.isDisplayed());
+        softAssertObject.assertTrue(rocketText.isDisplayed());
+
+        //SoftAsserting that the displayed texts match the expected texts
+        softAssertObject.assertEquals(microscopeText.getText(), expectedTextMicroscope);
+        softAssertObject.assertEquals(headsetText.getText(), expectedTextHeadset);
+        softAssertObject.assertEquals(screenText.getText(), expectedTextScreen);
+        softAssertObject.assertEquals(rocketText.getText(), expectedTextRocket);
```

Comment:
Comparing two collection with the text

Reply:
I will use collections in the comparison